quinn-proto/src/token.rs

```diff
@@ -12,71 +14,103 @@ use crate::{
     Duration, SystemTime, RESET_TOKEN_SIZE, UNIX_EPOCH,
 };
 
-pub(crate) struct RetryToken {
+/// A retry token
```

Comment:
I think this commit has too much going on at once, it seems to have all of:

* API changes
* Logic changes
* Refactoring/adding comments

Would be nice to split this into smaller pieces that do one thing.

Reply:
Done